### DISCUSSION TITLE
Info assistant in production

### DISCUSSION THREAD

**Discussion by ediaz-caio** [microsoft/PubSec-Info-Assistant#887] [General]
Has anyone considered using the info-assistant in a production environment? If so how are you managing the code and changes etc?

**Comment by bjakems** [comment]
The Information Assistant is an open source accelerator.  There is additional effort to adhere to the specific company's production application requirements.  With the latest version of IA 1.2, a secure mode was introduced that is typically a piece that is required for production deployments by securing the infrastructure.
For a production application deployment, typically a company will leverage a CI/CD pipeline to deploy out the code changes.  I suggest practicing pushing out code updates from a Codespace to your environment better understand the steps required, then automate in a pipeline.

**Comment by lon-tierney** [comment]
I'd suggest you take a look at the [Production Considerations](https://github.com/microsoft/PubSec-Info-Assistant/blob/main/docs/deployment/considerations_production.md) documentation for a start, and CI/CD pipelines to promote builds across environments is a best practice. We test full new deployment on every code change as well as upgrade in place on PRs, though as @bjakems stated, you will need to understand your requirements and processes and adapt to them.
